admin_guide/preventing_node_placement.adoc

```diff
@@ -0,0 +1,101 @@
+= Preventing Specific Node Placement
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can prevent users from targeting specific nodes
+when scheduling pods using labels and node selectors, as well as the node
```

Comment:
more accurate: "As a cluster administrator, you can set policy to prevent certain roles from targeting single nodes for pod scheduling, as well as defining how users may control which groups of nodes their new pods may be scheduled on."